setup.py

```diff
@@ -248,7 +248,7 @@ def do_setup():
             'configparser>=3.5.0, <3.6.0',
             'croniter>=0.3.17, <0.4',
             'dill>=0.2.2, <0.3',
-            'flask>=0.12, <0.12.2',
+            'Flask<0.12.2,>=0.10.0',
```

Comment:
Is there any specific reason for `>=0.10.0`?

Both of them are going to install `0.12.1` version right? Let me know if I am missing something?